games-util/pyfa/pyfa-2.28.1.ebuild

```diff
@@ -0,0 +1,100 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+PYTHON_REQ_USE="sqlite"
+
+inherit desktop eutils python-single-r1 xdg-utils
```

Comment:
- May be dos2unix and possibly other classes now, not eutils (a lot of stuff got moved out of there recently, see the gentoo-dev mailing list)
- Use xdg if you can, may even be able to drop the postinst and prerm definitions below.

Reply:
replaced with xdg class and dropped the now redundant functions